sdk/resourcemanager/README.md

```diff
@@ -33,6 +35,71 @@ If you are an existing user of the older version of Azure management library for
 ```
 [//]: # ({x-version-update-end})
 
+The services available via `azure-resourcemanager` are listed as below:
+
+<details>
+<summary> Services list </summary>
+
+- App Services
+- Authorization
+- CDN
+- Compute
+- Container Instance
+- Container Registry
+- Container Services (AKS)
+- Cosmos DB
+- DNS
+- Event Hubs
+- Insight (Monitor)
+- Key Vault
+- Managed Identity
+- Network
+- Private DNS
+- Redis
+- Resources
+- Service Bus
+- Spring Cloud
+- SQL
+- Storage
+- Traffic Manager
+</details>
+
+<br/>
+In the case where you are interested in certain service above or the service not included in the wrapper package, you can choose to use the standalone package for each service. Those packages follow the same naming patterns and design principals. For example, the package for Media Services has the following artifact information.
+
+<!-- TODO: package update start -->
+```xml
+<dependency>
+  <groupId>com.azure.resourcemanager</groupId>
+  <artifactId>azure-resourcemanager-mediaservices</artifactId>
+  <version>1.0.0-beta.1</version>
+</dependency>
+```
+<!-- TODO: package package end -->
+
+After above configuration, the manager can be authenticated by following code:
+
+<!-- TODO: embedme -->
+```java
+// share the same http client and credential
+HttpClient httpClient = HttpClient.createDefault();
+TokenCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
+AzureProfile azureProfile = new AzureProfile(AzureEnvironment.AZURE);
+
+// init AzureResourceManager using azure-resourcemanager
+AzureResourceManager azureResourceManager = AzureResourceManager.configure()
+    .withHttpClient(httpClient)
+    .authenticate(tokenCredential, azureProfile)
+    .withDefaultSubscription();
+
+// init MediaservicesManager using azure-resourcemanager-mediaservices
+MediaservicesManager mediaservicesManager = MediaservicesManager.configure()
+    .withHttpClient(httpClient)
+    .authenticate(tokenCredential, azureProfile);
+```
+
+Please note that some of those standalone packages might have different release cycles compared to `azure-resourcemanager`. This is mostly because of various demands of different Azure services. See [Standalone Packages][standalone_packages] for a complete list of standalone packages with the API versions they are consuming.
```

Comment:
Suggest to remove first sentence, no need to confuse customers about the complex release process.

Reply:
removed.